File: app/src/main/java/com/lucagiorgetti/surprix/ui/mainfragments/surpriseList/BaseSurpriseRecyclerAdapterListener.java
package com.lucagiorgetti.surprix.ui.mainfragments.surpriseList;

import android.widget.ImageView;

public interface BaseSurpriseRecyclerAdapterListener {
    void onSurpriseDelete(int position);

    void onImageClicked(String imagePath, ImageView imageView, int placeHolderId);
}

File: app/src/main/java/com/lucagiorgetti/surprix/ui/mainfragments/surpriseList/BaseSurpriseRecyclerAdapterListener.kt
package com.lucagiorgetti.surprix.ui.mainfragments.surpriseList

import android.widget.ImageView

interface BaseSurpriseRecyclerAdapterListener {
    fun onSurpriseDelete(position: Int)
    fun onImageClicked(imagePath: String, imageView: ImageView, placeHolderId: Int)
}